application/src/main/kotlin/com/studentcenter/weave/application/meetingTeam/service/application/MeetingTeamGetByInvitationCodeApplicationService.kt

```diff
@@ -0,0 +1,38 @@
+package com.studentcenter.weave.application.meetingTeam.service.application
+
+import com.studentcenter.weave.application.common.exception.MeetingTeamExceptionType
+import com.studentcenter.weave.application.meetingTeam.port.inbound.MeetingTeamGetByInvitationCodeUseCase
+import com.studentcenter.weave.application.meetingTeam.service.domain.MeetingTeamDomainService
+import com.studentcenter.weave.application.meetingTeam.util.MeetingTeamInvitationService
+import com.studentcenter.weave.application.meetingTeam.vo.MeetingTeamInvitation
+import com.studentcenter.weave.domain.meetingTeam.entity.MeetingTeam
+import com.studentcenter.weave.support.common.exception.CustomException
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+class MeetingTeamGetByInvitationCodeApplicationService(
+    val meetingTeamDomainService: MeetingTeamDomainService,
+    val meetingTeamInvitationService: MeetingTeamInvitationService,
+) : MeetingTeamGetByInvitationCodeUseCase {
+
+    override fun invoke(invitationCode: UUID): MeetingTeam {
+        val meetingTeamInvitation: MeetingTeamInvitation? =
+            meetingTeamInvitationService.findByInvitationCode(invitationCode = invitationCode)
+
+        validateInvitationCodeExists(meetingTeamInvitation)
+            .let { return meetingTeamDomainService.getById(it.teamId) }
```

Comment:
p4) `?.let` 활용하는게 조금 더 코틀린스러운 것 같아요.
p4) 만료인지 원래 없는 코드인지 알 수 없기 때문에 문구를 바꿔야할 것 같아요!
```suggestion
       return meetingTeamInvitationService.findByInvitationCode(invitationCode = invitationCode)?.let {
           meetingTeamDomainService.getById(it.teamId)
       } ?: throw CustomException(
           type = MeetingTeamExceptionType.INVITATION_CODE_NOT_FOUND,
           message = "초대 링크를 찾을 수 없어요! 초대링크를 확인해주세요!"
       )
```

Reply:
좋습니다! 다만 초대링크를 확인해 달라고 하면 사용자 입장에서는 어떻게 해야 할 지 애매하다고 생각해서 아래와 같이 수정할게요!

>  message = "초대 링크를 찾을 수 없어요! 초대링크를 확인해주세요!"

message = "초대 링크를 찾을 수 없어요! 새로운 초대 링크를 통해 입장해 주세요!"